changelog/removePreludeAssert.dd

```diff
@@ -0,0 +1,3 @@
+Removed prelude assert for constructors & destructors
+
+The compiler used to insert an `assert(this,”null this”);` at the start of constructors & destructors. To trigger these asserts you had to construct or destruct an aggregate at the null memory location. This would crash if you were to access any members, which is required for a constructor or destructor to do anything useful other than logging (which would likely pick up the null pointer anyway).
```

Comment:
s/this,/this, /
s/you had to construct/one needed to construct/
s/if you were to access any members/upon any data member access/
s/ other than logging (which would likely pick up the null pointer anyway)//